linalg/blockoperator.cpp

```diff
@@ -28,11 +27,11 @@ BlockOperator::BlockOperator(const Array<int> & offsets)
      op(nRowBlocks, nRowBlocks),
      coef(nRowBlocks, nColBlocks)
 {
-   op = static_cast<Operator *>(NULL);
+   op = static_cast<Operator *>(nullptr);
```

Comment:
```suggestion
   op = nullptr;
```
No cast needed?

Reply:
removed in https://github.com/mfem/mfem/pull/4675/commits/ea041467ae1ad6d007ea2a6e7880fbfa448f50a7